py-polars/src/lazyframe.rs

```diff
@@ -569,7 +565,7 @@ impl PyLazyFrame {
             datetime_format,
             float_precision,
             delimiter: separator,
-            quote,
+            quote: quote_char,
```

Comment:
I guess `SerializeOptions` should also be updated `delimiter -> separator` and `quote -> quote_char`, right?